src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -214,16 +214,9 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, JObject par
                 case "Debugger.scriptParsed":
                     {
                         string url = args?["url"]?.Value<string>() ?? "";
-
-                        switch (url)
-                        {
-                            case var _ when url == "":
-                            {
-                                logger.LogTrace($"ignoring empty: Debugger.scriptParsed {url}");
-                                return true;
-                            }
-                        }
                         logger.LogTrace($"proxying Debugger.scriptParsed ({sessionId.sessionId}) {url} {args}");
+                        if (url == "")
```

Comment:
Isn't this same as the existing code? Logs a message, and return true if `url == ""`?

Reply:
yes, it is. I need to think about it a bit more